packages/mobile/src/exchange/saga.ts

```diff
@@ -260,11 +270,13 @@ export function* exchangeGoldAndStableTokens(action: ExchangeTokensAction) {
     )
 
     if (!txId) {
+      CeloAnalytics.track(CustomEventNames.missing_tx_id)
       Logger.error(TAG, 'No txId. Did not exchange.')
       return
     }
     yield call(sendAndMonitorTransaction, txId, tx, account)
   } catch (error) {
+    CeloAnalytics.track(CustomEventNames.exchange_failed, { error })
```

Comment:
Looks like you forgot to add this to CustomEventNames

Reply:
Thanks for catching this